infrastructure/quick-deploy/aws/armonik/parameters.tfvars

```diff
@@ -78,6 +78,20 @@ control_plane = {
 # Parameters of admin GUI
 # Put to null if we not want deploy it
 admin_gui = {
+  app = {
+    name  = "admin-app"
+    image = "125796369274.dkr.ecr.eu-west-3.amazonaws.com/armonik-admin-app"
+    tag   = "0.7.2"
```

Comment:
This is not a 0.7.2, is it ?

Reply:
For now it is, since we haven't release the new version of the GUI.